src/app/components/OptimizelyPageMetrics/index.test.jsx

```diff
@@ -192,4 +254,99 @@ describe('Optimizely Page Complete tracking', () => {
     expect(global.IntersectionObserver).toHaveBeenCalledTimes(1);
     expect(optimizely.track).toHaveBeenCalledTimes(1);
   });
+
+  it('should not send tracking event when pageComplete is false', async () => {
+    useOptimizelyVariation.mockReturnValue('variation_1');
+
+    const { container } = render(
+      <ContextWrap pageType={ARTICLE_PAGE} service="news" isAmp={false}>
+        <OptimizelyPageMetrics />
+      </ContextWrap>,
+    );
+
+    const element = container.getElementsByTagName('div')[0];
+    const observerInstance = getObserverInstance(element);
+
+    act(() => {
+      triggerIntersection({
+        changes: [{ isIntersecting: true }],
+        observer: observerInstance,
+      });
+    });
+
+    await Promise.resolve();
+
+    // still does this
+    expect(global.IntersectionObserver).toHaveBeenCalledTimes(1);
+    expect(optimizely.track).toHaveBeenCalledTimes(0);
+  });
+});
+
+describe('Optimizely combined tracking', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+  // rewrite
+  it('should do both things', async () => {
+    useOptimizelyVariation.mockReturnValue('variation_1');
+
+    const { container } = render(
+      <ContextWrap pageType={ARTICLE_PAGE} service="news" isAmp={false}>
+        <OptimizelyPageMetrics pageView pageComplete />
+      </ContextWrap>,
+    );
+
+    await waitFor(() => {
+      expect(optimizely.track).toHaveBeenCalledTimes(1);
+    });
+    const element = container.getElementsByTagName('div')[0];
+    const observerInstance = getObserverInstance(element);
+
+    act(() => {
+      triggerIntersection({
+        changes: [{ isIntersecting: true }],
+        observer: observerInstance,
+      });
+    });
+
+    await Promise.resolve();
+
+    expect(global.IntersectionObserver).toHaveBeenCalledTimes(1);
+    expect(optimizely.track).toHaveBeenCalledTimes(2);
+  });
+
+  // reqrite - not sure this is a valid test
+  it('should do both things when component used in two places', async () => {
+    useOptimizelyVariation.mockReturnValue('variation_1');
+
+    const { container } = render(
+      <ContextWrap pageType={ARTICLE_PAGE} service="news" isAmp={false}>
+        <main>
+          <OptimizelyPageMetrics pageComplete />
+        </main>
+        {/* pretty sure I'm not actually checking scroll depth */}
+        <OptimizelyPageMetrics pageView pageDepth />
+      </ContextWrap>,
+    );
+
+    await waitFor(() => {
+      expect(optimizely.track).toHaveBeenCalledTimes(1);
```

Comment:
Let's use this so that we know exactly which event is being fired when
```suggestion
      expect(optimizely.track).toHaveBeenCalledWith('...');
```

Reply:
If we are expecting the function to be called multiple times but with different parameters, then we can use this notation

```suggestion
      expect(optimizely.track).toHaveBeenNthCalledWith(1, 'page-views');
      expect(optimizely.track).toHaveBeenNthCalledWith(2, 'page-depth');
```

Reply:
As above, I've gone down a different route with OptimizelyPageMetrics/index.test.jsx. So this should no longer be applicable